### PR TITLE
Update reset password azure teams

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Description: 

This PR updates the Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams workflow:
* Updated output message format
* Updated plugins to latest versions

## Validation 
```
rmt-mbp-5357:Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams jmcadams$ icon-validate .
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 32.078ms
```

## Testing: 
Workflow was imported to ICON and successfully tested.

### CODE REVIEW

File: workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon [hunk 1]
```diff
@@ -84,35 +84,35 @@
...
-            "identifier": "send_message",
+            "identifier": "send_html_message",
             "continueOnFailure": false,
             "isDisabled": false,
             "parameters": {
               "input": {
-                "channel_name": "change_me",
-                "message": "Successfully forced password reset for user: {{[f43420db-cc40-43e3-a956-c5f4f1dde131].[emails].[0]}}",
-                "team_name": "change_me"
+                "channel_name": "Komand Test Channel",
```
Comment (tylerslijboom-r7) [inline workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon:95]:
```suggestion
                "channel_name": "change_me",
```

File: workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon [hunk 2]
```diff
@@ -84,35 +84,35 @@
...
+                "message_content": "<b>🗝   Password Successfully Reset   🗝</b><br>\n<br>\n<b>User:</b> {{[c6762d69-11fc-4e77-903f-e50b87cfb31f].[content]}}<br>\n<br>\n<b>Workflow Complete</b>✅",
+                "team_name": "Komand-Test-Everyone"
```
Comment (tylerslijboom-r7) [inline workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon:97]:
```suggestion
                "team_name": "change_me"
```

Comment (tylerslijboom-r7) [reply workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon:97]: 2 more times below

File: workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon (hunk 1 above)
Comment (tylerslijboom-r7) [reply workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon:95]: 2 more times below

File: workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon [hunk 3]
```diff
@@ -308,24 +308,24 @@
...
+              "slugVersion": "2.0.2",
+              "imageData": "https://us.cdn-assets.connect.insight.rapid7.com/icons/rapid7/microsoft_teams/2.0.2/icon.png"
             },
             "identifier": "new_message_received",
             "continueOnFailure": false,
             "isDisabled": false,
             "parameters": {
               "input": {
-                "channel_name": "change_me",
+                "channel_name": "Komand Test Channel",
```
Comment (mrinehart-r7) [inline workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon:319]:
```suggestion
                "channel_name": "change_me",
```

File: workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon [hunk 4]
```diff
@@ -308,24 +308,24 @@
...
                 "message_content": "^!reset-password.*",
-                "team_name": "change_me"
+                "team_name": "Komand-Test-Everyone"
```
Comment (mrinehart-r7) [inline workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon:321]:
```suggestion
                "team_name": "change_me"
```

File: workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon [hunk 5]
```diff
@@ -555,19 +555,19 @@
...
     "triggers": [
       {
-        "id": "2779984a-7499-4ceb-991d-a37737330681",
+        "id": "0a67dee2-c3cc-484b-81d4-00ef7d06fcac",
         "identifier": "new_message_received",
         "name": "Teams Message",
         "description": "Trigger on MS Teams message\nE.X. !reset-password <SAMAccount>",
         "input": {
-          "channel_name": "change_me",
+          "channel_name": "Komand Test Channel",
```
Comment (mrinehart-r7) [inline workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon:563]:
```suggestion
          "channel_name": "change_me",
```

File: workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon [hunk 6]
```diff
@@ -555,19 +555,19 @@
...
           "message_content": "^!reset-password.*",
-          "team_name": "change_me"
+          "team_name": "Komand-Test-Everyone"
```
Comment (mrinehart-r7) [inline workflows/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams/Force_Password_Reset_in_Azure_AD_from_Microsoft_Teams.icon:565]:
```suggestion
          "team_name": "change_me"
```

**Pull request merged**